lib/presentation/crowdaction/crowdaction_details/widgets/confirm_participation.dart

```diff
@@ -147,39 +154,39 @@ class _ConfirmParticipationState extends State<ConfirmParticipation> {
                         widget.onSelect(selectedIds);
                       },
                     ),
-                    const SizedBox(
-                      height: 20,
-                    ),
-                    PillButton(
-                      text: "Confirm Participation",
-                      isLoading: state is SubscribingToCrowdAction,
-                      isEnabled: _commitments.isNotEmpty,
-                      onTap: () {
-                        // TODO - Confirm Participation
-                        context.read<SubscriptionBloc>().add(
-                              SubscriptionEvent.participate(
-                                widget.crowdAction,
-                                _commitments,
-                                "",
-                              ),
-                            );
-                      },
-                      margin: EdgeInsets.zero,
-                    ),
-                    const SizedBox(height: 20),
-                    SizedBox(
-                      width: double.infinity,
-                      height: 52,
-                      child: TextButton(
-                        onPressed: () {
-                          Navigator.pop(context);
-                        },
-                        child: const Text("Cancel"),
-                      ),
-                    ),
                   ],
                 ),
               ),
+              const SizedBox(
+                height: 20,
+              ),
+              PillButton(
+                text: "Confirm Participation",
+                isLoading: state is SubscribingToCrowdAction,
+                isEnabled: _commitments.isNotEmpty,
+                onTap: () {
+                  // TODO - Confirm Participation
```

Comment:
We don't need to confirm participation? It's not a harmful action that cannot be undone. Should we remove this TODO or leave it for later deliberation?